configman/value_sources/for_modules.py

```diff
@@ -189,10 +196,40 @@ def local_to_str(a_thing):
 
 
 #==============================================================================
+class OrderableObj(object):
+    """Python3 can't sort non-string types implicitly.
+    """
+    def __init__(self, value):
+        if not isinstance(value, six.string_types):
+            value_str = to_str(value)
+        else:
+            value_str = value
+        self.value_str = value_str
+        self.value = value
+    def __lt__(self, other):
+        return (self.value_str < other.value_str)
+    def __repr__(self):
+        return self.value
+
+
+#==============================================================================
+class OrderableTuple(object):
+    """Python3 can't sort non-string types implicitly.
+    """
+    def __init__(self, value, index=1):
+        self.value = value
+        self.index = index
+    def __lt__(self, other):
+        return (self.value[self.index] < other.value[self.index])
+    def __repr__(self):
+        return self.value
```

Comment:
Done.  If this doesn't work, let's revisit https://github.com/mozilla/configman/pull/163#issuecomment-229910455 .